gui/src/pages/Job.svelte

```diff
@@ -0,0 +1,22 @@
+<script lang="ts">
+  import Layout from '../components/Layout.svelte';
+  import JobTree from '../components/JobTree.svelte';
+  import { api } from '../lib/api';
+
+  export let params = { job: '' };
+
+  const jobId = params.job;
+
+  const tasks = api.kernel.describeTasks({ jobIds: [jobId] });
+</script>
+
+<Layout showBackButton backButtonRoute="#/jobs">
+  <!-- TODO: common loader component -->
```

Comment:
Maybe wrap everything within `Layout` here inside a `<section>...</section>` that has `min-height: 100vh;` and `background: #ffffff` or wait until #131 lands and use `<MonoPanel>...</MonoPanel>`

Reply:
Going to wait for #131 - this UI is expectedly ugly as sin still.